spec/order_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'order'
+
+describe Order do
+  describe '#add_dish' do
+
+    it 'starts with an empty basket' do
+      order = Order.new
+
+      expect(order.basket).to eq []
```

Comment:
This test is just testing state so you don't need it. We know Order will be initialised with an empty array that's what you're doing every time you create a new order. A test that would test behaviour rather than state would be using add_dish and making sure the array now has a dish in it (which you're doing in the test below).